tests/mal_scraper/test_anime.py

```diff
@@ -36,5 +36,21 @@ def test_download_first(mock_requests):
 
 def test_download_first_fail(mock_requests):
     """Do we get None if the page was bad??"""
-    mock_requests.add('http://myanimelist.net/anime/1', fake='bla')
+    mock_requests.always_mock(
+        'http://myanimelist.net/anime/1',
+        'garbled_anime_page',
+    )
     assert mal_scraper.retrieve_anime(1) is None
+
+
+class TestParsing:
+    """Test parsing of particular parts of the page"""
+
+    def test_unknown_episodes(self, mock_requests):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/QasimK/mal-scraper/file/2650471559/issues/source?bid=3298861&fileBranchId=3324330#l49)